src/menu.c

```diff
@@ -2872,7 +2878,7 @@ s32 func_80087F14(s32 *controllerIndex, s32 arg1) {
         sControllerPakBadData[i] = 0;
         ret = get_free_space(i, &bytesFree, &notesFree);
         pakStatus = ret & 0xFF; //Upper 2 bits are controller index
-        if (ret == 0) {
+        if (ret == CONTROLLER_PAK_GOOD) {
```

Comment:
It's annoying how GitHub doesn't let you leave comments outside of where deltas are, but do you want to change the return type to `SIDeviceStatus`?

Reply:
Sure, I'll do that. Good call. It's not completed yet, but it seems pretty reasonable that it's enough done to know that at least.